sdks/python/apache_beam/typehints/typehints.py

```diff
@@ -507,7 +507,7 @@ def __getitem__(self, py_type):
       raise TypeError('An Option type-hint only accepts a single type '
                       'parameter.')
 
-    return Union[py_type, None]
+    return Union[py_type, type(None)]
```

Comment:
<!--new_thread; commit:3e97dd3f1feff22c5467c0ebd6226c2763406962; resolved:1-->
This looks fine to me, but I want to check with @robertwb whether there is a subtlety I may be missing here.